PowerEditor/src/NppDarkMode.cpp

```diff
@@ -1184,9 +1520,9 @@ namespace NppDarkMode
 		SetWindowSubclass(hwnd, GroupboxSubclass, g_groupboxSubclassID, pButtonData);
 	}
 
-	constexpr UINT_PTR g_tabSubclassID = 42;
+	constexpr UINT_PTR g_comboBoxSubclassID = 42;
```

Comment:
There will be 2 same values (42) for both g_tabSubclassID and g_comboBoxSubclassID.
It seems fine for the compiled PR. However, will it have some risk to have any impact?

Reply:
No, every ID there - for any control - is 42.